### ISSUE TITLE
Add vers

### ISSUE THREAD

**Issue by ZloeSabo** [DmitryKoterov/dklab_realsync#32]
https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#basic-instructions

**Comment by hempalex** [comment]
I agree. I made my own formula https://github.com/hempalex/homebrew-formula  but i will be easier to use official homebrew repository
